README.md

```diff
@@ -17,95 +18,110 @@ This module adds a new exec provider capable of executing PowerShell commands.
 
 ##Module Description
 
-Puppet provides a built-in `exec` type that is capable of executing commands. This module adds a `powershell` provider to the `exec` type,  which enables all of the `exec` parameters, such as `creates`, `onlyif`, `unless`, etc. This module is particularly helpful if you need to run PowerShell commands but don't know the details about how PowerShell is executed, since you can technically run PowerShell commands in Puppet without the module.
+Puppet provides a built-in `exec` type that is capable of executing commands. This module adds a `powershell` provider to the `exec` type,  which enables `exec` parameters, listed below. This module is particularly helpful if you need to run PowerShell commands but don't know the details about how PowerShell is executed, since you can technically run PowerShell commands in Puppet without the module.
```

Comment:
There's already a link in the ToC to the reference, so a link is not necessary here. We're trying to avoid excessive linking in the readmes. The "listed below" was added in conjunction with the removal of the link to exec's documentation to help clarify we're specifically listing relevant parameters in the readme.